### PR TITLE
Order of arguments changed in nfs export command, BZ#2032858

### PR DESCRIPTION
**Pull request by hkadam134**
BZ#2032858
Order of arguments has changed from build - 16.2.7

Logs : http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-XIJD0G/

Reference Bz : https://bugzilla.redhat.com/show_bug.cgi?id=2032858

Signed-off-by: Hemanth <hyelloji@redhat.com>

# Description

Please include Automation development guidelines. Source of Test case - New Feature/Regression Test/Close loop of customer BZs
<details>

<summary>click to expand checklist</summary>

- [ ] Create a test case in Polarion reviewed and approved.
- [ ] Create a design/automation approach doc. Optional for tests with similar tests already automated.
- [ ] Review the automation design
- [ ] Implement the test script and perform test runs
- [ ] Submit PR for code review and approve
- [ ] Update Polarin Test with Automation script details and update automation fields
- [ ] If automation is part of Close loop, update BZ flag qe-test_coverage “+” and link Polarion test
</details>

### CODE REVIEW

File: tests/cephfs/nfs-ganesha_basics.py [hunk 1]
```diff
@@ -49,8 +49,8 @@ def run(ceph_cluster, **kw):
             # Create cephfs nfs export
             nfs_client[0].exec_command(
                 sudo=True,
-                cmd=f"ceph nfs export create cephfs {fs_name} {nfs_name} "
-                f"{nfs_export_name} path={path}",
+                cmd=f"ceph nfs export create cephfs {nfs_name} "
```
Comment (psathyan) [inline tests/cephfs/nfs-ganesha_basics.py:52]: This is specific to 5.1 only or does it work for 5.0 also

Comment (hkadam134) [reply tests/cephfs/nfs-ganesha_basics.py:52]: Yes, It's applicable only on 5.1, I am working on the same thing.. Will update in some time

**Comment by hkadam134** [comment]
5.0 Logs : http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-TWU7QO/
5.1 Logs : http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-LQ37JW/

File: tests/cephfs/nfs-ganesha_basics.py [hunk 2]
```diff
@@ -47,11 +47,19 @@ def run(ceph_cluster, **kw):
             else:
                 raise CommandFailed("Failed to create nfs cluster")
             # Create cephfs nfs export
-            nfs_client[0].exec_command(
-                sudo=True,
-                cmd=f"ceph nfs export create cephfs {fs_name} {nfs_name} "
-                f"{nfs_export_name} path={path}",
-            )
+            if "5.1" in rhbuild:
```
Comment (psathyan) [inline tests/cephfs/nfs-ganesha_basics.py:50]: Could you check `if "5.0" in rhbuild` instead of `5.1`?

Comment (hkadam134) [reply tests/cephfs/nfs-ganesha_basics.py:50]: Ok

**Review by psathyan — APPROVED** [review]

**Pull request merged**